lib/slack/rtm.ex

```diff
@@ -42,6 +42,6 @@ defmodule Slack.Rtm do
 
   defp slack_url(token) do
     Application.get_env(:slack, :url, "https://slack.com") <>
-      "/api/rtm.start?token=#{token}&batch_presence_aware=true&presence_sub=true"
+      "/api/rtm.connect?token=#{token}&batch_presence_aware=true&presence_sub=true"
```

Comment:
If we change `start` to `connect` here, could that cause issues for people who are still relying on the old `start` behavior?

If that's the case, I wonder if it makes sense to make a new `connect` method that uses `rtm.connect` while keeping the `start` method that keeps using `rtm.start`?

Reply:
I just noticed that my bot was fully broken, and `rtm.start` _always_ returned a 429 rate limit error, even with a single request. I interpreted that as Slack turning off the API.

Perhaps that behavior differs for older Bots though?

Reply:
Yes this was fundamentally the issue I ran into with #177 (which lead to #184 and #190)